scio/src/main/scala/com/spotify/featran/scio/package.scala

```diff
@@ -18,19 +18,37 @@
 package com.spotify.featran
 
 import com.spotify.scio.values.SCollection
+import org.tensorflow.{example => tf}
 
 import scala.reflect.ClassTag
 
+
 package object scio {
 
   /**
    * [[CollectionType]] for extraction from Scio `SCollection` type.
    */
   implicit object ScioCollectionType extends CollectionType[SCollection] {
     override def map[A, B: ClassTag](ma: SCollection[A], f: (A) => B): SCollection[B] = ma.map(f)
+
     override def reduce[A](ma: SCollection[A], f: (A, A) => A): SCollection[A] = ma.reduce(f)
+
     override def cross[A, B: ClassTag](ma: SCollection[A],
                                        mb: SCollection[B]): SCollection[(A, B)] = ma.cross(mb)
+
```

Comment:
`-`

Reply:
@ravwojdyla  all of this is Intellij code format feature. Do you have different settings or do you not use it?